app/presenters/mainstream_browse_page_presenter.rb

```diff
@@ -9,6 +9,28 @@ def format
   def links
     super.merge(
       "related_topics" => @tag.topics.order(:title).map(&:content_id),
+      "active_top_level_browse_page" => top_level_browse_page_id,
+      "top_level_browse_pages" => @tag.class.sorted_parents.map(&:content_id),
+      "second_level_browse_pages" => second_level_browse_pages,
     )
   end
+
+private
+
+  def top_level_browse_page_id
+    if @tag.has_parent?
+      [@tag.parent.content_id]
+    else
+      []
```

Comment:
Did we not want to populate `active_top_level_browse_page` for a top-level page?